### PR TITLE
add configurable service annotations

### PR DESCRIPTION
**Pull request by erohmensing**

### CODE REVIEW

File: charts/rasa-x/templates/rasa-services.yaml [hunk 1]
```diff
@@ -7,6 +7,10 @@ metadata:
   labels:
     {{ include "rasa-x.labels" $ | nindent 4 }}
     app.kubernetes.io/component: {{ .serviceName }}
+{{- with .Values.rasa.service.annotations }}
```
Comment (tczekajlo) [inline charts/rasa-x/templates/rasa-services.yaml:10]: The rasa service is a bit tricky
```suggestion
{{- with $.Values.rasa.service.annotations }}
```

Comment (erohmensing) [reply charts/rasa-x/templates/rasa-services.yaml:10]: Ha, I'd just figured this out 🙈  thanks!

**Comment by erohmensing** [comment]
Question: How does the event service interact with rabbit and the db if it doesn't have a service? 🤔

**Comment by tczekajlo** [comment]
> Question: How does the event service interact with rabbit and the db if it doesn't have a service? 🤔

Through a service :D We use external helm charts to install postgresql and rabbitmq, and those helm charts create services. You have to check external helm charts if there is support for extra annotations for services that they create.

**Comment by erohmensing** [comment]
oh for sure, those shouldn't be a problem, I meant the eventService itself, whether it needs to expose itself, but i guess it is consuming rabbit via polling, not being posted to, and is only writing to the db, so no services actually need to know where it lives?

**Comment by tczekajlo** [comment]
> oh for sure, those shouldn't be a problem, I meant the eventService itself, whether it needs to expose itself, but i guess it is consuming rabbit, not being posted to, and is only writing to the db, so no deployments actually need to know where it lives?
Aa, I see, yes. The event-service doesn't need a service. Exactly as you described, other deployments don't need to know where it lives.

**Comment by erohmensing** [comment]
[Rabbit](https://github.com/bitnami/charts/blob/master/bitnami/rabbitmq/values.yaml#L614)
[Postgres](https://github.com/bitnami/charts/blob/master/bitnami/postgresql/values.yaml#L353
[Redis](https://github.com/bitnami/charts/blob/master/bitnami/redis/values.yaml#L376)

So i believe we should be ready to go ✅

**Review by tczekajlo — APPROVED** [review]
🚀

**Pull request merged**